README.md

```diff
@@ -9,7 +9,7 @@ The ROCm™ Data Center Tool simplifies the administration and addresses key inf
 
 For up-to-date document and how to start using RDC from pre-built packages, please refer to the [**ROCm DataCenter Tool User Guide**](https://rocm.docs.amd.com/projects/rdc/en/latest/)
 
-For certificate generation, please refer to [**RDC Developer Handbook**#generate-files-for-authentication](https://rocm.docs.amd.com/projects/rdc/en/latest/user_guide/handbook.html#generate-files-for-authentication)
+For certificate generation, please refer to [**RDC Developer Handbook**#generate-files-for-authentication](https://rocm.docs.amd.com/projects/rdc/en/latest/install/handbook.html#generate-files-for-authentication)
```

Comment:
new link does not work, old one still does:
https://rocm.docs.amd.com/projects/rdc/en/latest/user_guide/handbook.html#generate-files-for-authentication

Reply:
ahh I see, those pages will be created after this gets merged?